src/System.Drawing.Common/ref/System.Drawing.Common.csproj

```diff
@@ -1,16 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{D7AEA698-275D-441F-B7A7-8491D1F0EFF0}</ProjectGuid>
     <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
-    <Configurations>net461-Debug;net461-Release;netcoreapp-Debug;netcoreapp-Release;netfx-Debug;netfx-Release;netstandard-Debug;netstandard-Release;uap-Debug;uap-Release</Configurations>
+    <Configurations>netcoreapp-Debug;netcoreapp-Release;</Configurations>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Drawing.Common.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsNetFx)' == 'true' OR '$(TargetsNETStandard)' == 'true'">
     <Compile Include="System.Drawing.Common.netstandard.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsNetCoreApp)' == 'true' OR '$(TargetsUap)' == 'true'">  
+  <ItemGroup Condition="'$(TargetsNetCoreApp)' == 'true' OR '$(TargetsUap)' == 'true'">
     <Compile Include="System.Drawing.Common.Forwards.cs" />
```

Comment:
We can remove this and Uncondition `IsPartialFacadeAssembly`

Reply:
We no longer need this manual forwards if you set `IsPartialFacadeAssembly` to true.

Reply:
I'm confused. `IsPartialFacadeAssembly` was only on for `NetFx`. Are you saying we should always have it on now? Why are we able to do that, if so?

Reply:
I think it was because of the way the ref assembly was composed, but now that we are building it live only for 3.0 and uap, we can now use IsPartialFacadeAssembly, but no need to fix it. Either way works fine.